---
title: "Final Report: Dietary Yogurt Influence on Human Fitness and Disease"
author: "Brian Rezende"
date: "December 5, 2018"
output:
github_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Computational

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

## Subsections are ok in the results section too

```{r your-code-here, echo=FALSE}
# Add code chunks as needed for your analyses
# For most analyses, I would recommend splitting the
# intensive computational part into a seperate R script file
# and then just load your libraries and the data object here
# using the `load()` function. Ask me for clarification if this is
# unclear.
```

```{r load-packages-and-data, echo=FALSE, warning=FALSE}
# load packages as needed
library("dplyr")
library("tidyr")
library("citr")
library("knitr")
library("ggplot2")
library("phyloseq")

# load data produced from analysis scripts using
# something like load("output/processed_data.Rdata")
load("output/phyloseq_obj.R")
load("output/melted_obj.R")
```

```{r most-abundant-phylum-day-0, warning=FALSE, echo=FALSE}
melted_phyloseq %>%
  filter(timepoints == "0") %>%
  filter(!is.na(Genus)) %>%
  group_by("Genus") %>%
  ggplot(aes(x = Genus,
             y = Abundance,
             fill = Subject))+
  geom_bar(stat = "Identity")+
  xlab("Genus")+
  ylab("Abundance")+
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1))+
  ggtitle("Abundance of Bacteria on Day 0 per Subject")
  
```

**Figure 1:** 

```{r most-abundant-phylum-day-7, echo=FALSE}
melted_phyloseq %>%
  filter(timepoints == "7") %>%
  filter(!is.na(Genus)) %>%
  group_by("Genus") %>%
  ggplot(aes(x = Genus,
             y = Abundance,
             fill = Subject))+
  geom_bar(stat = "Identity")+
  xlab("Genus")+
  ylab("Abundance")+
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1))+
  ggtitle("Abundance of Bacteria on Day 7 per Subject")

```

**Figure 2:**

```{r most-abundant-phylum-day-42, echo=FALSE}
melted_phyloseq %>%
  filter(timepoints == "42") %>%
  filter(!is.na(Genus)) %>%
  group_by("Genus") %>%
  ggplot(aes(x = Genus,
             y = Abundance,
             fill = Subject))+
  geom_bar(stat = "Identity")+
  xlab("Genus")+
  ylab("Abundance")+
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1))+
  ggtitle("Abundance of Bacteria on Day 42 per Subject")
```
 
**Figure 3:**

```{r plot-richness-across-day-0-7-42-with-abundance, echo=FALSE}
# Plot richness within each subject observed during each check point of the
# study. The Shannon index was used to look at species richness and evenness. 
plot_richness(phyloseq_obj,
              x = "Subject",
              color = "Day",
              measures = c("Observed", "Shannon")) +
  xlab("Subject") +
ggtitle("Plot richness of subjects throughout the study")
```

*Figure 4:* Plot richness of each subject with comparisons of observed and Shannon index. There is an observed spread amongst each of the day samples were collected. Overlaps indicate no change or regression back to the original condition with some variation. Alpha diversity measure was used for observed and a smaller measure was seen in the Shannon index. 


```{r general-abundance-within-subjects, echo=FALSE}
# This figure allows us to draw comparisons between each subject and track chan
# ges over time in regards to abundance of bacterial communities. 
melted_phyloseq %>%
  ggplot(aes(x = Subject,
             y = Abundance,
             fill = timepoints)) +
  geom_col(position = position_dodge()) +
  xlab("Subject") +
  ylab("Abundance") +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Total Abundance in Each Subject")
```

*Figure 5*: Total abundance of bacteria found in each subject, according to the date that their fecal specimen was collected and tested. Subject E showed no instance of bacteria or samples upon initially entering into the study. Subject F left before the first week due to dislike of the yogurt diet.

```{r boxplot-of-subject-with-treatment, echo=FALSE}
melted_phyloseq %>%
  ggplot(aes(x = Treatment,
             y = Abundance,
             fill = Phylum)) +
  geom_boxplot() +
  xlab("Treatment") +
  ylab("Abundance") +
  ggtitle("Scaled Boxplot of Treatment Effects on Different Phylum Abundance") +
  theme_bw() +
  scale_y_log10() + # Use this function to observe orders of magnitude or when
  # samples are too clustured together to observe differences
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1))
```

*Figure 3:* The two types of treatment and its effects on the presence of the different phylum and how abundance changes over the course of the study. Outliers are only present within the probiotic treatment group. Greater averages are observed within the treatment group as well with the exception of Actinobacteria. 

```{r observed-plot-richness-with-alpha-diversity, echo=FALSE}
# Data frame is small enough to refer to "observed" and it demonstrates that
# averages were around the same for each time point.
plot_richness(phyloseq_obj, x = "timepoints", measures = "Observed") +
  geom_boxplot()
```

*Figure 4:* Alpha diversity measure at three distinct time points, which reveals that observed data is uniform throughout the study with noticeable outliers at each timepoint. There is a greater diversity seen at the day 42 timepoint in comparison to the other time points. The boxplot demonstrates evenness as well throughout the 3 time points. 

```{r filtered-ABCDG-treatment, echo = FALSE}
melted_phyloseq %>%
  filter(Subject %in%
           c("Subject A", "Subject B", "Subject C", "Subject D",
             "Subject G")) %>%
  ggplot(aes(x = Subject,
             y = Abundance,
             fill = Treatment)) +
  geom_line() +
  xlab("Subjects") +
  ylab("Abundance") +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Abundance  vs Control") +
  theme_bw()
```

```{r-compare-makeup-of-bacterial-community-abundance-with-abundance}
melted_phyloseq %>%
  ggplot(aes(x = Subject,
             y = Abundance,
             fill = Genus)) +
  geom_col(position = position_dodge()) +
  xlab("Subject") +
  ylab("Abundance") +
  theme(axis.text.x = element_text(angle = 90,
                                   hjust = 1)) +
  ggtitle("Total Abundance in Each Subject")
```

Shannon-Wiener index provides a good estimate of ecological diversity, which allowed us to quantify the changes in gut microbial diversity. This diversity was based on terminal restriction fragments (T-RFs) and NGS universal bacteria primer data after the patients ate yogurt. Shannon index takes into account both richness and evenness of each group. The first two weeks suggested a trend towards greater diversity, followed by a plateau and drop by the time it reached day 42. 


# Discussion

We observe changes in the microbial community composition begins to change by 7 days of yogurt consumption. Clustering is then observed at days 14 and 42. This vast gap between sample collection dates could possibly hold implications for natural therapeutics. 
Variable results arise from the independent choice of primers and sequencing methods (Fouhy et al). 


# Sources Cited
